worldengine/generation.py

```diff
@@ -130,11 +130,32 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
             ('plain', hl),
             ('hill', ml),
             ('mountain', None)]
+    armonize_ocean(ocean, e, ocean_level)
     world.set_ocean(ocean)
     world.set_elevation(e, e_th)
     world.sea_depth = sea_depth(world, ocean_level)
 
 
+def armonize_ocean(ocean, elevation, ocean_level):
+    """
+    The goal of this function is to make the ocean floor less noisy.
+    The underwater erosion should cause the ocean floor to be more uniform
+    """
+    width, height = elevation.shape
+
+    shallow_sea = ocean_level * 0.85
```

Comment:
this should be the elevation which corresponds to the sea level. Then we could have depressions which have an elevation below the sea level but are land. sea_depth would normally be equal to `sea_level - elevation`, of course only for actual sea/ocean tiles.

You made me realize that we are also touching tiles which are not ocean... fixing.